checkout/src/main/java/com/checkout/tokenization/repository/TokenRepositoryImpl.kt

```diff
@@ -55,9 +57,13 @@ internal class TokenRepositoryImpl(
                 NonCancellable,
         )
 
+    @VisibleForTesting
+    var dispatcher = Dispatchers.IO
```

Comment:
Let's avoid having an mutable variable here, and move it to the constructor.

We could do it like this 

```
    private val riskSdkUseCase: RiskSdkUseCase,
    private val dispatcher: CoroutineDispatcher = Dispatchers.IO,
) : TokenRepository {
```